control/vector_field_waypoint.hpp

```diff
@@ -56,7 +57,7 @@ extern "C"
  */
 typedef struct
 {
-    const Mavlink_waypoint_handler*   waypoint_handler;         ///< Waypoint list (input)
+    Mavlink_waypoint_handler*   waypoint_handler;         ///< Waypoint list (input)
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/331/commits/c578caff4fd67489951f710e4a7a653ea895b461